clouds/clouds-aws/src/test/java/aws/cloudformation/di/suitetests/AwsEapStandaloneManagementClientTests.java

```diff
@@ -0,0 +1,59 @@
+package aws.cloudformation.di.suitetests;
+
+
+import aws.core.identification.AwsEc2Instance;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.wildfly.extras.creaper.core.online.CliException;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import sunstone.api.EapMode;
+import sunstone.api.Parameter;
+import sunstone.api.StandaloneMode;
+import sunstone.api.WithAwsCfTemplate;
+
+import java.io.IOException;
+
+import static aws.cloudformation.AwsTestConstants.eapMngmtPassword;
+import static aws.cloudformation.AwsTestConstants.eapMngmtPort;
+import static aws.cloudformation.AwsTestConstants.eapMngmtUser;
+import static aws.cloudformation.AwsTestConstants.instanceName;
+import static aws.cloudformation.AwsTestConstants.region;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+/**
+ * Shared resources among other DI tests.
```

Comment:
If you will improve this in other test. Check this out as well.

Reply:
Fixed.